srcpkgs/jetbrains-jdk-11/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'jetbrains-jdk-11'
+
+_jdk_build=520.2
+_jdk_fullver="11.0.4"
+_archive_name="jbrsdk-${_jdk_fullver//\./_}-linux-x64-b${_jdk_build}.tar.gz"
+
+pkgname=jetbrains-jdk-11
+version="${_jdk_fullver}b${_jdk_build}"
+revision=1
+archs="x86_64"
+wrksrc="jbrsdk"
+hostmakedepends="aria2"
+short_desc="JetBrains Java 11 JDK"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="GPL-2.0+CE"
+homepage="https://github.com/JetBrains/JetBrainsRuntime"
+distfiles="https://dl.bintray.com/jetbrains/intellij-jbr/${_archive_name}"
+checksum=690307a7bde2ae6ee11f245d8064bfa928bb6b03240ce75caebcb6002c934472
+nopie=yes
+# This JDK appears to link to libs that do not exist, but still continues to function well even in their absence.
+# Best guess is that they are optional. ¯\_(ツ)_/¯
+noverifyrdeps=yes
+
+fetch_cmd="aria2c -c"
```

Comment:
There are some redirects there that `wget` doesnt seem to follow correctly, while `aria2` does.
Since this is only for the build machines (right?), is it a big issue that `aria2` is left as the fetch command?

Reply:
This seems to work with `wget` for me now, so switched to it.